includes/compat/class-wc-stripe-email-failed-refund.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Base class for the email sent to the admin and customers when a refund fails.
+ *
+ * @since 9.6.0
+ */
+abstract class WC_Stripe_Email_Failed_Refund extends WC_Email_Failed_Order {
+	/**
+	 * Returns the list of template parameters.
+	 *
+	 * @return array
+	 */
+	abstract public function get_template_params();
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->title   = __( 'Stripe refund failed', 'woocommerce-gateway-stripe' );
+		$this->heading = __( 'Refund failed', 'woocommerce-gateway-stripe' );
+		$this->subject = __( '[{site_title}] Refund failed for #{order_number}.', 'woocommerce-gateway-stripe' );
```

Comment:
After seeing this in the emails, I think we should adjust this slightly so it is more aligned with the other emails we are sending. The changes are to use `order #{order_number}` and to remove the trailing `.`.
```suggestion
		$this->subject = __( '[{site_title}] Refund failed for order #{order_number}', 'woocommerce-gateway-stripe' );
```

Reply:
```suggestion
		$this->subject = __( '[{site_title}]: Refund failed for #{order_number}', 'woocommerce-gateway-stripe' );
```

For alignment we can put a `:` after the site name.